customize.dist/translations/messages.es.js

```diff
@@ -18,7 +18,7 @@ define(function () {
     out.synchronizing = "Sincronización";
     out.reconnecting = "Reconectando...";
     out.lag = "Retraso";
-    out.readonly = 'Solo lectura';
+    out.readonly = 'Sólo lectura';
```

Comment:
This translation was intended as `es_ES`. Language variants are a bit difficult to deal with, since it's not always clear what the differences are, and making a culture invariant version of a translation is, in my experience, almost impossible. The compromise I've seen is to either not offer invariants, or to default the invariant to the most common variant. Take the case of `en_US`/`en_GB`: how do you choose between US spellings (color) or GB spellings (colour)? 🤔